### PR TITLE
Prevent eip deletion

### PR DESCRIPTION
**Pull request by swbsf**
[move const to types definition](https://github.com/quortex/kubestatic/commit/8ff8dbda4d986c757b1de8651cc32af67d1d434a)
[Add PreventEIPDeallocation spec + handle EIP disassociation through tags](https://github.com/quortex/kubestatic/commit/de023c10aba9133d267c7ca54831423719790f41)

### CODE REVIEW

**Review by vincentmrg — APPROVED** [review]

**Review by vincentmrg — DISMISSED** [review]

File: helm/kubestatic/Chart.yaml [hunk 1]
```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.2
+version: 0.4.0-beta.2
```
Comment (HansCeril) [inline helm/kubestatic/Chart.yaml:18]: 0.5.0

File: helm/kubestatic/Chart.yaml [hunk 2]
```diff
@@ -15,10 +15,10 @@ type: application
...
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: 0.3.3
+appVersion: 0.4.0-beta.2
```
Comment (HansCeril) [inline helm/kubestatic/Chart.yaml:24]: 0.5.0

File: helm/kubestatic/Chart.yaml [hunk 3]
```diff
@@ -15,10 +15,10 @@ type: application
...
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.2
+version: 0.5.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: 0.3.3
+appVersion: 0.5.0
```
Comment (antonincms) [inline helm/kubestatic/Chart.yaml:24]:
```suggestion
appVersion: 0.5.0

```

Comment (vincentmrg) [inline helm/kubestatic/Chart.yaml:24]:
```suggestion
appVersion: 0.5.0

```

**Pull request merged**